napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -403,7 +405,9 @@ def change_preserve_labels(self, state):
         state : QCheckBox
```

Comment:
Optional: similar to https://github.com/napari/napari/pull/5541/files#r1100890632, this docstring should probably be updated.